src/controlnet_aux/processor.py

```diff
@@ -11,7 +11,7 @@
                             LeresDetector, LineartAnimeDetector,
                             LineartDetector, MediapipeFaceDetector,
                             MidasDetector, MLSDdetector, NormalBaeDetector,
-                            OpenposeDetector, PidiNetDetector, ZoeDetector)
+                            OpenposeDetector, PidiNetDetector, ZoeDetector,)
```

Comment:
dwpose support should be added

Reply:
DWPose is built on a series of MM libraries, which may lead to version conflicts with other dependencies. So I think it is better to treat it as an optional dependency.

Reply:
Added.